beaker.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'minitar', '~> 0.5.4'
-  s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'json', '~> 1.4'
```

Comment:
Looks like this is sort-of documented on https://github.com/bundler/bundler/issues/4571 - but remains inconsistent with what documentation seems to imply.